docs/pfcwd/README_PFCWD_Various_Reboot_Scenarios.md

```diff
@@ -0,0 +1,100 @@
+# Readme PFCWD Various Reboot Scenarios
+- [Readme PFCWD Various Reboot Scenarios](#readme-pfcwd-various-reboot-scenarios)
+  - [Background](#background)
+    - [PFC storm detection](#pfc-storm-detection)
+    - [PFC storm mitigation](#pfc-storm-mitigation)
+    - [PFC storm restoration](#pfc-storm-restoration)
+    - [PFC watchdog implementation](#pfc-watchdog-implementation)
+  - [PFC watchdog commands on SONiC](#pfc-watchdog-commands-on-sonic)
```

Comment:
please move these sections to a general readme

Reply:
Omitted the TOC section from readme